### PR TITLE
Add Supriya to team site

### PR DESCRIPTION
**Pull request by lucypotato**

### CODE REVIEW

**Comment by lucypotato** [comment]
~~Waiting on confirmation from Gregor that Supriya's title is Web Computing Specialist.~~

Confirmed :thumbsup:

File: public/index.html [hunk 1]
```diff
@@ -504,7 +525,7 @@ <h4>Lucy Yang</h4>
                     <!-- member name & designation -->
                     <div class="member-meta">
                         <h4>Sarah Yatsko</h4>
-                        <span>Interim Director</span>
+                        <span>Director</span>
```
Comment (lucypotato) [inline public/index.html:528]: Not sure why this change is in here, it was from a previous PR that got merged. I thought I had pulled the latest from master before starting this new branch. Will it be an issue?

Comment (keithligon) [reply public/index.html:528]: I don't think so, looks like the last [commit](https://github.com/cssat/cssat_site/commit/1a97b6047652f5408c4ac3f2707ff03706ba343b) updated team.yml.

Comment (lucypotato) [reply public/index.html:528]: Yea and it looks like this branch has no conflicts with the base branch too.

**Review by keithligon — APPROVED** [review]
lgtm!
